### DISCUSSION TITLE
Open RPG Linter Configuration is Missing

### DISCUSSION THREAD

**Discussion by thomprl** [codefori/vscode-ibmi#1994] [Q&A] [answered]
I asked this on Ryver as well but thought I'd post here.  Some reason my Open RPG Linter Configuration is missing when I right click on a filter.  If I try to open it via F1 it says is was not found. I'm running the latest extensions.

![image](https://github.com/codefori/vscode-ibmi/assets/17936568/e97b7d0a-beab-4e80-806a-bf929d1fae5e)

![image](https://github.com/codefori/vscode-ibmi/assets/17936568/f582e5fd-8153-47de-a722-cd6764e079a0)

**Comment by worksofliam** [comment]
You can run the command when you have a member open. I will look into why it's not showing up in the Object Browser view.

**Reply by SusanSiD** [reply]
fwiw, I've got the same problem. Since I'm teaching this on Wednesday, can you confirm for me - when you say "you can run the command when you have member open" - do you mean "RPGLE: Open RPGLE lint configuration" ?  I want to make sure I've got it right before telling a few hundred folks in a few days! Thanks.

**Reply by thomprl** [reply, marked as answer]
It worked for me.  Open an RPG program and then press F1 and type RPG and you should see this option.

![image](https://github.com/codefori/vscode-ibmi/assets/17936568/c24779d5-6116-447f-9970-6fca7abcb806)

**Reply by SusanSiD** [reply]
Thanks @thomprl. I had done that and it did work. I just wanted to double-check that was the "best answer" before putting it on charts for my webinar. I'm just glad to hear I'm not the only one experiencing this problem. I usually figure that I've done something wrong since I still feel like a newb with VS Code.

**Reply by worksofliam** [reply]
I have just released vscode-rpgle 0.26.6 which fixes the issue where the right click option was not shown. Thanks!

**Reply by SusanSiD** [reply]
I just updated and it all seems to work fine now. Thanks, Liam.

**Comment by thomprl** [comment]
Thanks Liam..
